robottelo/vm.py

```diff
@@ -56,13 +51,22 @@ def __init__(
             self, cpu=1, ram=512, distro=None, provisioning_server=None,
             image_dir=None, tag=None, hostname=None, domain=None,
             target_image=None):
+        distro_el6 = settings.distro.image_el6
+        distro_el7 = settings.distro.image_el7
+        base_images = [distro_el6, distro_el7]
         self.cpu = cpu
         self.ram = ram
-        self.distro = BASE_IMAGES[-1] if distro is None else distro
-        if self.distro not in BASE_IMAGES:
+        if distro == 'rhel6':
+            distro = distro_el6
+        if distro == 'rhel7':
+            distro = distro_el7
+        if distro is None:
+            distro = distro_el7
+        self.distro = distro
```

Comment:
I just found this approach as updating it by other ways was too much of a task, which I felt was not desired.
I feel updating each and every DISTRO_RHEL7 and DISTRO_RHEL6 is too much of a task.
So, I am  just updating the settings value in the VirtualMachine __init__ function itself.

so with least changes, everything would work smoothly.